main.py

```diff
@@ -1,8 +1,39 @@
 from fastapi import FastAPI
+import requests
+import random
 
 app = FastAPI()
 
 
+# Homepage. Directs you to '/facts' page or '/breeds' page, where either you get a cat fact pulled from the
+# catfact.ninja 'facts' get request, or a breed from the 'breeds' get request"
+# https://catfact.ninja/#/
 @app.get("/")
 async def root():
-    return {"message": "Hello World"}
+    return "Random fact (from a list) generated at '/facts'. Random breed generated at '/breeds'."
+
+
+# Returns a random cat fact from a list of facts (default length of 10 facts)
+@app.get("/facts")
+async def root():
+    # URL to send an HTTP get request to
+    resp = requests.get("https://catfact.ninja/facts").json()
```

Comment:
What if the request fails, what will json() return? Will it fail or return a error message? Please investigate that. My suggestion is check the response status-code. Read this https://requests.readthedocs.io/en/latest/user/quickstart/#response-content. 
It will be smth like:
```
resp = requests.get("https://catfact.ninja/facts")
if resp.status_code == 200:
  return resp.json()
else:
  return {"error": resp.reason, "status_code": resp.status_code}

```

Reply:
@rafaelbrant addressed in new commit, line 23 & 42